tests/test_ext_intersphinx.py

```diff
@@ -229,3 +234,53 @@ def test_getsafeurl_unauthed():
     expected = 'https://domain.com/project/objects.inv'
     actual = _get_safe_url(url)
     assert expected == actual
+
+
+def test_debug_noargs(capsys):
+    """debug interface, without arguments"""
+    with pytest.raises(SystemExit):
+        debug(['sphinx/ext/intersphinx.py'])
+
+    expected = dedent("""
+        Print out an inventory file.
+        Error: must specify local path or URL to an inventory file.
+    """).strip()
+    stdout, stderr = capsys.readouterr()
+    assert stdout == ""
+    assert stderr == expected + "\n"
+
+
+def test_debug_file(capsys, tempdir):
+    """debug interface, with file argument"""
+    inv_file = tempdir / 'inventory'
+    inv_file.write_bytes(inventory_v2)
+
+    debug(['sphinx/ext/intersphinx.py', str(inv_file)])
+
+    stdout, stderr = capsys.readouterr()
+    assert stdout.startswith("c:function\n")
+    assert stderr == ""
+
+
+@mock.patch('requests.get')
+def test_debug_url(fake_get, capsys):
```

Comment:
it looks like this test confirms `fetch_inventory()`.
I feel mocking does not help to test for `debug()`.